src/components/02_atoms/Message/Message.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { css } from 'emotion';
-import { string, oneOf } from 'prop-types';
```

Comment:
Are we going to do this all over the place now? Maybe we can make an issue to migrate to this usage of `PropTypes`

Reply:
>  Maybe we can make an issue to migrate to this usage of PropTypes

Reply:
https://github.com/jsdrupal/drupal-admin-ui/issues/160